app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,128 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      A modern languages teacheer opens young minds to the languages and cultures around the world. </p>
+      <p>As we strive to achieve our vision for a 'Global Britain' mutual inter-cultural understanding and insights through the fostering of a shared language, is vital.</p>
+      <p>Qualifications are available in 18 modern languages with the must popular being French, German and Spanish in key stage 2 and 3.</p>
+  <p>Tax-free scholarships of £27,000 and bursaries of £25,000 are available for eligible trainee language teachers.</p>
+  </section>
+
+  <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "",
+      colour: "white",
+      image_1_path: "static/images/content/hero-images/0017.jpg",
+      image_1_alt: "",
+      image_2_path: "static/images/content/hero-images/0024.jpg",
+      image_2_alt: "",
+    ) do %>
+  
+    <% end %>
+  </section>
+
+    <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "As a language teacher, you can light the flame of language learning and build cultural capital for all young people",
+    name: "Teacher",
+    background: "grey",
+    large: true
+  ) %>
+</section>
+
+  <section class="col col-720 col-horizontal col-space-m">
+    <div>
+      <h2>What you'll be teaching</h2>
+      <p>You will follow the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-languages-progammes-of-study">national curriculum. For any modern language you teach, you will:</p>
+<ul>
+  <li>prepare lessons and teaching materials </li>
```

Comment:
@caitlin-reidy these bullet points seem a bit obvious. Most people would assume as a teacher you'd have to prepare lessons and mark homework e.t.c. So I'm not sure we need to include any of these bullet points as they don't really add anything? We don't have these on the other subject pages.